lte/gateway/docker/s1ap/readme.md

```diff
@@ -71,3 +45,35 @@ If inside of container, CTRL+d or exit from container and run stop script
 root@472f8708ec12:/magma/lte/gateway/python/integ_tests# exit
 s1ap/stop-s1ap.sh
 ```
+
+# Simple example
+
+```
+sudo -i
```

Comment:
Can you explain the necessity to execute all commands as root user?
It would probably be good to identify the commands needing root permissions, by specifically calling them with `sudo`.

Reply:
The [AGW install script checks for sudo](https://github.com/magma/magma/blob/master/lte/gateway/deploy/agw_install_docker.sh#L21-L25), and for simplicity in this example I just ran them all as sudo.